build/yamls/audit-config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Config
+preferences: {}
+clusters:
+- name: example-cluster
+  cluster:
+    server: http://10.96.0.100:8080
```

Comment:
I think we can try using the DNS name here instead of the ClusterIP, even though it may not work for everyone. Can you try with `webhook-service.default.svc`?